### PR TITLE
consistently return - instead of None

### LINKED ISSUE #408: ioc get all defaults deviate from default set on jail creation
the `defaults` use `None` in several places, where a freshly created jail uses `-`

```sh
% ioc get all defaults | grep None
id:None
jail_zfs_dataset:None
```

### PR DESCRIPTION
**Pull request by igalic**
the baseconfig now too returns "-" instead None
This fixes #408

### CODE REVIEW

File: iocage/lib/Config/Jail/BaseConfig.py [hunk 1]
```diff
@@ -150,7 +150,7 @@ def _set_legacy(  # noqa: T484
             self.legacy = False
 
     def _get_id(self) -> str:
-        return str(self.data["id"])
+        return iocage.lib.helpers.to_string(self.data["id"])
```
Comment (igalic) [inline iocage/lib/Config/Jail/BaseConfig.py]: oops

```
./iocage/lib/Config/Jail/BaseConfig.py:153:10: T484 Returning Any from function declared to return "str"
```

File: iocage/lib/Config/Jail/BaseConfig.py [hunk 2]
```diff
@@ -382,7 +382,7 @@ def _set_vnet(  # noqa: T484
 
     def _get_jail_zfs_dataset(self) -> typing.List[str]:
         try:
-            jail_zfs_dataset = str(self.data["jail_zfs_dataset"])
+            jail_zfs_dataset = iocage.lib.helpers.to_string(self.data["jail_zfs_dataset"])
```
Comment (igalic) [inline iocage/lib/Config/Jail/BaseConfig.py]: double oops

```
./iocage/lib/Config/Jail/BaseConfig.py:385:80: E501 line too long (90 > 79 characters)
./iocage/lib/Config/Jail/BaseConfig.py:386:14: T484 Returning Any from function declared to return "List[str]"
```

File: iocage/lib/Config/Jail/BaseConfig.py (hunk 1 above)
Comment (gronke) [reply iocage/lib/Config/Jail/BaseConfig.py]: Just wrap `str()` around it - that should satisfy the type check.

**Comment by igalic** [comment]
well, in that case… *closes pr*

**Pull request closed without merging**